integration-tests/pages/errorPage.js

```diff
@@ -1,6 +1,6 @@
 const page = require('./page')
 
-const errorPage = () => page('Sorry, there is a problem with the service', {})
+const errorPage = () => page('Page not found', {})
```

Comment:
this looks like a strange change - don't we need a `notFoundPage.js` too?

Reply:
will check and confirm - will pull new error handling to separate PR